docs/sphinx/dev_guide/ci.rst

```diff
@@ -8,13 +8,76 @@
 
 .. _ci:
 
-**********************
-Continuous Integration
-**********************
+************************************
+Continuous Integration (CI) Testing
+************************************
+
+The RAJA project employes multiple tools to run its tests for each GitHub
```

Comment:
I don't think you meant to include the French word for employee lol
(typo on employ)

Reply:
Fixed.